app/websocket/app/main/events.py

```diff
@@ -43,8 +43,13 @@ def cmd(message):
     """ Forward pysyft command to hook virtual worker. """
     try:
         worker = hook.local_worker
-        if not len(worker.current_objects()):
-            recover_objects(hook)
+        # mari-linhares: I've commented these lines
+        # due to issues storing plans on the database.
+        # Maybe there are too many bytes being stored
+        # but more exploration is needed.
+        # TODO: uncomment these lines
+        # if not len(worker.current_objects()):
+        #     recover_objects(hook)
```

Comment:
@IonesioJunior any suggestions on how to deal with this? I think we can just check the types before storing in the database, what do you think?

Reply:
sounds a good idea to me